src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -55,6 +55,8 @@
     <!-- We do not have a training scenario that covers lib/netstandard2.0/System.Text.Encoding.CodePages.dll  -->
     <ExpectedDependency Include="System.Text.Encoding.CodePages" />
     <ExpectedDependency Include="System.Numerics.Vectors"/>
+    <ExpectedDependency Include="Microsoft.Extensions.DependencyInjection" />
```

Comment:
I don't think we're expecting to insert these into VS are?  Or will we be the first ones to need this and therefore ship it?

Reply:
I don't really understand the whole "ExpectedDependency" thing because we don't do it in Razor, but the build failed without this addition. Is there somewhere else we should be adding these if they're already inserted into VS?

Reply:
I believe these are dependencies that we _insert_ into VS.  I don't think we should be inserting this package so I believe it should be in the list starting at line 101 in this file.